packages/teams-js/src/public/geoLocation.ts

```diff
@@ -0,0 +1,132 @@
+import { sendAndHandleSdkError as sendAndHandleError } from '../internal/communication';
+import { ensureInitialized } from '../internal/internalAPIs';
+import { errorNotSupportedOnPlatform, FrameContexts } from './constants';
+import { DevicePermission, ErrorCode } from './interfaces';
+import { runtime } from './runtime';
+
+/**
+ * Namespace to interact with the geoLocation module-specific part of the SDK. This is the newer version of location module.
```

Comment:
Just a question -- is this capability still under development and should all namespaces, functions, interfaces, enums, etc. in this file have a @beta tag?

Reply:
How do we decide on that? It is done on teams-js side. On host side, if some host have not implemented the interface, that can be checked with isSupported. 
@TrevorJoelHarris

Reply:
Added @beta tags